app/components/rental.hbs

```diff
@@ -0,0 +1,29 @@
+<article class='rental'>
+  <!--
+    Note, below, the Rental::Image (double colon) syntax is the Namespaced Syntax
+    https://guides.emberjs.com/release/tutorial/part-1/more-about-components/#toc_organizing-code-with-namespaced-components 
+  -->
```

Comment:
Nice quick reference for myself to refer back to when I set myself loose to experiment with my own solo Ember app after this tutorial. Also just typing things out like this helps my brain memorize it faster.